client/webserver/site/src/html/settings.tmpl

```diff
@@ -16,6 +16,18 @@
         [[[Show pop-up notifications]]]
       </label>
     </div>
+    <div {{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+    <span>Fiat Currency: </span><span id="fiatCurrency">{{.FiatCurrency}}</span>
+    </div>
+    <div id="fiatRateSources" {{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+    <span class="mb-2">[[[fiat_exchange_rate_sources]]]:</span>
+    {{range $source, $enabled := .FiatRateSources}}
```

Comment:
> Fallback? Do they all get requested and only one used?

They all get requested and an average fiat rate is used as the final rate for assets, if any expires/stops to return rates, others are used as a fallback. Do users need to read it somewhere? 

> Which does it use for, say BTC-USD?

<img width="247" alt="Screenshot 2022-07-09 at 1 09 08 AM" src="https://user-images.githubusercontent.com/57448127/178083755-26d0b891-027b-4447-abb8-dbab2586c4bf.png">

`Fiat Currency` shows the fiat rate currency `USD` e.g(BTC-USD, ETH-USD, DCR-USD), any thoughts on the display?

Reply:
> They all get requested and an average fiat rate is used as the final rate for assets, if any expires/stops to return rates, others are used as a fallback if enabled. Do users need to read it somewhere?

They need some kind of explanation that rates are averaged if on multiple sources.  That's not obvious.  I can see an "i" in circle with a tooltip, as short as possible.

> Fiat Currency shows the fiat rate currency USD e.g(BTC-USD, ETH-USD, DCR-USD), any thoughts on the display?

My question "Which does it use for, say BTC-USD?" was meant to show a user's thought process.  Say I'm trading something against BTC (or just withdrawing BTC), which source has that exchange rate?  Do they all have the same asset rates available or do some have assets that others do not?  If the idea is that they will all have the same info, just different values, OK, but that's not a given.

e.g. Does dcrdata really provide exchange rates for LTC and BCH?  If I want rates for these, what sources do I pick?

Reply:
I can live with this as-is if the general guidance is something like: "Sources may provide fiat exchange rates for different subsets of assets.  You should select all sources that are acceptable to get the fiat exchange rates for the most assets.  Assets with fiat rates from multiple sources will use the average from all of those sources."

Is that generally accurate?  We should revise the wording, but I'm trying to understand how it's meant to be interpreted.

Reply:
@buck54321 any thoughts on this? I agree users need to know what fiat exchange rates are available per source, and a clear UI interpretation of this information is important.

Reply:
We don't have to go down the rabbit hole to Lodi them all now.  I just want to understand.
If my understanding is right, let's just put a little text in a tooltip, otherwise we just end up answering the question in chat and GitHub issues.

Reply:
> @buck54321 any thoughts on this? I agree users need to know what fiat exchange rates are available per source, and a clear UI interpretation of this information is important.

It's not too bad and it doesn't have to be perfect right away. The new tooltip is good enough for now.